teachertool/src/components/RubricWorkspace.tsx

```diff
@@ -101,7 +106,7 @@ function getActionMenuItems(tab: TabName): MenuItem[] {
     return items;
 }
 
-const WorkspaceToolbarButtons: React.FC = () => {
+const WorkspaceToolbarButtons: React.FC<PrintRefProps> = ({ printRef }) => {
```

Comment:
Lets keep with this naming pattern for consistency.

```suggestion
const WorkspaceToolbarButtons: React.FC<WorkspaceToolbarButtonsProps> = ({ printRef }) => {
```

Reply:
So, I didn't want to make different prop interfaces for these because they have the same prop. I don't know the benefit of giving both the toolbarbuttons and tabpanels different prop interfaces if we aren't going to make them unique in the future, which I can't imagine a scenario where either of them would need other props.

Reply:
If, down the line, we needed to add a new property to one of these components, the developer responsible for making this change would need to pause and understand the reasoning behind sharing this interface, and evaluate whether it is ok to split it off into a new one. They will think through this carefully. Is this future cognitive load justified here? I'm concerned we're degrading the orthogonality of the design without good reason. Sharing a property name isn't a good justification for a base class.